oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBuilder.java

```diff
@@ -170,6 +174,8 @@ public class DocumentNodeStoreBuilder<T extends DocumentNodeStoreBuilder<T>> {
     private boolean clusterInvisible;
     private boolean throttlingEnabled;
     private boolean fullGCEnabled;
+    private Set<String> fullGCIncludePaths = of();
+    private Set<String> fullGCExcludePaths = of();
```

Comment:
btw, just a side-comment, I find this less readable than say `Set.of()` (but I think this might be controversial as IDEs visualize it magically - but if you read the raw file it is less readable for me)

Reply:
fixed in https://github.com/apache/jackrabbit-oak/pull/1551/commits/55f310e2d65cfe3208ff7688ce93787eccb0cfba